src/test/java/org/javarosa/core/model/actions/OdkNewRepeatEventTest.java

```diff
@@ -188,12 +189,12 @@ public void newRepeatInstance_canUsePreviousInstanceAsDefault() throws IOExcepti
         scenario.next();
         scenario.next();
         scenario.answer(7);
-        assertThat(scenario.answerOf("/data/repeat[0]/q"), is(intAnswer(7)));
+        assertThat(scenario.answerOf("/data/repeat[1]/q"), is(intAnswer(7)));
 
         scenario.next();
         scenario.createNewRepeat();
         scenario.next();
-        assertThat(scenario.answerOf("/data/repeat[1]/q"), is(intAnswer(7)));
+        assertThat(scenario.answerOf("/data/repeat[2]/q"), is(intAnswer(7)));
```

Comment:
This was previously checking the first node a second time rather than verifying that the second node started out with a value of 7 before then being changed to 8.